docs/get-started/roles-and-permissions/access_roles.md

```diff
@@ -0,0 +1,161 @@
+# Access roles in PMM
+
+Roles are a vital part of Access control. Roles provide users with access to specific, role-based metrics.
+
+## Enabling access roles in PMM
+
+You can enable access roles in PMM as follows:
+
+**Docker**
+
+To enable access roles in a ``pmm-server`` docker container, pass an additional environment variable ``RBAC_ENABLE=1`` when starting the container.
+
+```sh
+docker run … RBAC_ENABLE=1
```

Comment:
This should also be `ENABLE_RBAC`, it might be good to add the `-e ` to suggest that it's an env variable.

```sh
docker run ... -e ENABLE_RBAC=1 ...
```

Reply:
Fixed.